---
layout: page
title: 开源工具
menubar: false
# hero_image: https://www.restoringautoimmunehealth.com/wp-content/uploads/2019/01/DNA-image-1080x640.jpg
hero_image: /baojunwang/assets/dnabackground.jpg
show_sidebar: false
---



## GENETIC PARTS

[**BioPartsDB**]( http://biopartsdb.bio.ed.ac.uk/)<br>
BioParts Database (for biocircuit design)

[**Registry of Standard Biological Parts**](http://partsregistry.org/Main_Page)<br>
Public registry of standard biological parts; mostly for iGEM

[**Addgene**](http://www.addgene.org/Syn_Bio/Voigt/Terminators/)<br>Commercial collection of plasmids.

[**SynBERC Registry**](https://registry.synberc.org/#page=login)<br>Public registry of biological parts; mostly for SynBERC members. 

[**JBEI Registry**](https://registry.jbei.org/#page=login)<br>Public registry of standard biological parts. 

 

## SEQUENCE DESIGN
[**RBS Library Calculator**](https://salis.psu.edu/software/RBSLibraryCalculatorSearchMode) <br>Uniform sampling of protein expression for efficient combinatorial optimization. Courtesy of Salis Lab (Penn State).

[**RBS Generator**](http://github.com/hsalis/ribosome-binding-site-calculator)<br>Source code for RBS Calculator.

[**ApE**](http://www.biology.utah.edu/jorgensen/wayned/ape/)<br>A fairly simple but very useful (free) plasmid editor software download.

[**GeneDesign 3.0**](http://54.235.254.95/gd/)<br>One-stop shopping for gene construction tools (random sequence generator, reverse translation, etc).

[**Optimizer**](http://genomes.urv.es/OPTIMIZER/)<br>Reverse translation with "Guided random" option: outputs an optimized coding sequence that uses all codons in the organism (not just the most common).

[**Gene2Oligo**](http://berry.engin.umich.edu/gene2oligo/)<br>Very useful when you need to assemble a sequence from individual oligos.

[**Random DNA Generator**](http://www.faculty.ucr.edu/~mmaduro/random.htm)<br>Generates a random DNA sequence with defined GC content

[**VectorNTI**](http://www.lifetechnologies.com/us/en/home/life-science/cloning/vector-nti-software.html)<br>Comprehensive proprietary software for DNA design.

[**Geneious**](http://www.geneious.com/)<br>Proprietary bioinformatics software for DNA design.

[**NuPack**](http://www.nupack.org/)<br>Growing software suite for the analysis and design of nucleic acid systems.  

[**Reverse Complement**](http://www.bioinformatics.org/sms/rev_comp.html) <br>Converts a DNA sequence into its reverse, complement, or reverse-complement counterpart

[**Integrated Microbial Genomes**](http://img.jgi.doe.gov/cgi-bin/pub/main.cgi) <br>Comparatively analyze genome data  by gene, genome and function

## NUCLEOTIDE SEQUENCE ANALYSIS
[**Mfold**](http://mfold.rna.albany.edu/?q=mfold)<br>RNA and DNA folding algorithms

[**UNAfold**](http://mfold.rna.albany.edu/) <br>RNA and DNA folding algorithms

[**Oligocalc**](http://www.basic.northwestern.edu/biotools/oligocalc.html)<br>RNA and DNA oligo properties.

[**Finnzymes Tm Calculator**](https://www.finnzymes.fi/tm_determination.html)<br>DNA melting temp calculator for Phusion PCR(nearest-neighbor method)

[**DNA/RNA/Protein Molecular Weight Calculator**](http://www.changbioscience.com/genetics/mw.html)<br>Calculates mass of single- and double-stranded DNA and RNA.

[**Rare Codon Calculator**](http://nihserver.mbi.ucla.edu/RACC/)<br>Outputs graphical information of the codon usage in your sequence.

[**Graphical Codon Usage Analyser**](http://gcua.schoedl.de/)<br>Calculates codon usage statistics from a DNA sequence

[**Restriction Mapper**](http://www.restrictionmapper.org/)<br>Finds restriction sites in a DNA sequence

[**Microbes On-line**](http://www.microbesonline.org/)<br>Genome analysis tools

[**Online Analysis Tools**](http://molbiol-tools.ca/)<br>Another portal to good tools.  

## PROTEIN SEQUENCE ANALYSIS
[**Protein Calculator**](http://www.scripps.edu/~cdputnam/protcalc.html)<br>Protein Properties

[**SignalP**](http://www.cbs.dtu.dk/services/SignalP/)<br>Signal peptide cleavage site predictor

[**ProTISA**](http://mech.ctb.pku.edu.cn/protisa/)<br>Database of confirmed prokaryotic translation initiation sites

[**ExPASy**](http://ca.expasy.org/tools/) <br>Comprehensive proteomics toolbox

## ALIGNMENT TOOLS
[**LALIGN**](http://www.ch.embnet.org/software/LALIGN_form.html)<br>Basic sequence alignment tool

[**BLAST: Basic Local Alignment Search Tool**](http://blast.ncbi.nlm.nih.gov/Blast.cgi)<br>Protein and nucleotide sequence database alignment tool

[**Mobyl@pasteur**](http://mobyle.pasteur.fr/cgi-bin/portal.py?form=clustalw-multialign)<br>Sequence Alignment and Analysis (ClustalW)  

## CYTOMETRY ANALYSIS
[**FlowJo**](http://www.flowjo.com/)<br>Our preferred proprietary cytometry data analysis tool.

[**Cyflogic**](http://www.cyflogic.com/)<br>Free cytometry data analysis software

[**FCSExtract Utility**](http://research.stowers-institute.org/efg/ScientificSoftware/Utility/FCSExtract/index.htm)<br>Converts \*.\*fcs files to \*.\*csv files  

## INFORMATION PORTALS
[**E. coli Statistics**](http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi)<br>Useful informatin on **E. coli**

[**E. coli genotypes in OWW**](http://openwetware.org/wiki/E._coli_genotypes)<br>Best collection of *E. coli* genotypes and their meanings.

[**EcoCyc E. coli Database**](https://ecocyc.org/)<br>
Encyclopedia of *E. coli* Genes and Metabolism.

[**BioNumbers**](http://bionumbers.hms.harvard.edu/)<br>Database of useful biological numbers.

[**Plasmid Databases**](http://www.science.co.il/Biomedical/Plasmid-Databases.asp)<br>Links to various plasmid sequence resources

[**EcoliWiki**](http://ecoliwiki.net/colipedia/index.php/Welcome_to_EcoliWiki)<br>Portal for *E. coli* research.

[**Cold Springs Harbor Protocols**](http://cshprotocols.cshlp.org/)<br>Journal of laboratory protocols  

## OTHER CALCULATORS
[**BioMath Calculators**](http://www.promega.com/biomath/)<br>Useful unit converstions

[**Ammonium Sulfate Calculator**](http://www.encorbio.com/protocols/AM-SO4.htm)<br>Computes the amount of solid ammonium sulfate required to reach a given percent saturation

[**Centrifugation Rotor Calculator**](http://www.changbioscience.com/cell/rcf.html)<br>Interconverts RPM and RCF for common rotors  

[**Gibson Calculator**](http://voigtlab.ucsf.edu/ruby/assembly) <br>Primer design for Gibson ligations (KT)

[**Hill Coefficient Calculator**](http://voigtlab.ucsf.edu/ruby/hill) <br>Calculates Hill coefficient from tab-delineated data

## PRESENTATION TOOLS
[**UCSF Chimera Home Page**](https://www.cgl.ucsf.edu/chimera/)<br>Fantastic Molecular imaging tool

[**Instructions for making a movie in Chimera**](http://www.cgl.ucsf.edu/chimera/data/tutorials/movies09/moviemaking.html)<br>Super fast and easy.

[**Adobe Illustrator**](http://www.adobe.com/products/illustrator.html)<br>Stop using ugly, ad hoc Powerpoint graphics! Make gorgeous vector images in Illustrator instead!

[**Bring on the box plots**](http://blogs.nature.com/methagora/2014/01/bring-on-the-box-plots-boxplotr.html)

[**BoxPlotR**](http://boxplot.tyerslab.com/)<br>A web-tool for generation of box plots  <> 

[**Jalview**](www.jalview.org)<br> One of the most widely used applications for visualising and analysing DNA, RNA and protein multiple sequence alignments. It is designed to enable anyone working with protein, RNA and DNA sequences to investigate their structure, function and evolution, and create high-quality figures for publication. Open source, developed by Dundee Univ.  

## Other Useful Resources & Tools

[**Global Core Biodata Resources**](https://globalbiodata.org/scientific-activities/global-core-biodata-resources/)


## Journal Sites

[**Nature**](https://www.nature.com/)   [**Nature Biotechnology**](https://www.nature.com/nbt/)   [**Nature Chemical Biology**](https://www.nature.com/nchembio/)   [**Nature Methods**](https://www.nature.com/nmeth/)   [**Nature Microbiology**](https://www.nature.com/nmicrobiol/)   [**Nature Communications**](https://www.nature.com/ncomms/)   [**Nature Biomedical Engineering**](https://www.nature.com/natbiomedeng/)

[**Science**](https://www.sciencemag.org/)   [**Science Translational Medicine**](https://stm.sciencemag.org/)   [**Science Advances**](https://advances.sciencemag.org/)    [**Cell**](https://www.cell.com/)   [**Cell Host & Microbe**](https://www.cell.com/cell-host-microbe/home)   [**Molecular Cell**](https://www.cell.com/molecular-cell/home)   [**Cell Systems**](https://www.cell.com/cell-systems/home)   [**Molecular Systems Biology**](http://msb.embopress.org/)   [**eLife**](https://elifesciences.org/)

[**PNAS**](https://www.pnas.org/)   [**Nucleic Acids Research**](https://academic.oup.com/nar)   [**ACS Synthetic Biology**](https://pubs.acs.org/journal/asbcd6)   [**Analytical Chemistry**](https://pubs.acs.org/journal/ancham)   [**Biosensors & Bioelectronics**](https://www.journals.elsevier.com/biosensors-and-bioelectronics)   [**ACS Sensors**](https://pubs.acs.org/journal/ascefj)   [**JACS**](https://pubs.acs.org/journal/jacsat)   [**Advanced Materials**](https://onlinelibrary.wiley.com/journal/15214095)   [**AFM**](https://onlinelibrary.wiley.com/journal/16163028)
