<html>
 <head>
  <meta charset="utf-8"/>
  <title>
   Browse by Publication
  </title>
  <link href="../styles/styling.css" rel="stylesheet"/>
  <link href="https://fonts.googleapis.com/css?family=Lato|Questrial" rel="stylesheet"/>
  <link href="../images/logo_square.png" rel="icon"/>
 </head>
 <nav>
    <ul>
     <li style="float: left">
      <a href="../browsing/Home.html">
       <img src="../images/logo.png" style="height: 70%; vertical-align: middle; margin: -20 0;"/>
      </a>
     </li>
     <li style="float: left">
      <a href="../browsing/Home.html">
       Home
      </a>
     </li>
     <li style="float: left" class="dropdown">
      <a href="../browsing/Catalogue.html">
       Browse
      </a>
      <div class="dropdown-content">
       <a href="../browsing/BrowsebyType.html">
        By Type
       </a>
       <a href="../browsing/BrowsebyFunction.html">
        By Function
       </a>
       <a href="../browsing/BrowsebyPublication.html">
        By Publication
       </a>
      </div>
     </li>
     <li style="float: left">
      <a style="white-space: nowrap;" href="https://github.com/fxbuson/biopartsDB">
       Help File
      </a>
     </li>
     <li style="float: right">
      <a href="../browsing/search.html">
       Search
      </a>
     </li>
    </ul>
   </nav>
 <body>
  <div class="page-content">
   <div class="full_column">
    <h1>
     Browse by Publication
    </h1>
    <p>
     Some labs characterize a noteworthy amount of publications that fit the standards of the repository.
                    Some others characterize a large amount of parts within some few publications.
                    These are organized here by the PI leading the research group
                    for each of the characterizations.
    </p>
    <h2>
     Wang Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Wang,B2013.html">
       <p>
        A modular cell-based biosensor using engineered genetic logic circuits to detect and integrate multiple environmental signals.
       </p>
      </a>
      <p>
       Wang, B., Barahona, M. and Buck, M., in 2013
      </p>
      <p>
       <strong>
        4 Regulated Promoters
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Liu,Y2019.html">
       <p>
        Engineered CRISPRa enables programmable eukaryote-like gene activation in bacteria
       </p>
      </a>
      <p>
       Liu, Y., Wan, X., and Wang, B., in 2019
      </p>
      <p>
       <strong>
        6 CRISPR Systems
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Pinto,F2020.html">
       <p>
        An expanded library of orthogonal split inteins enables modular multi-peptide assemblies
       </p>
      </a>
      <p>
       Pinto, F., Thornton, E. and Wang, B., in 2020
      </p>
      <p>
       <strong>
        15 Split Iteins
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/type_reporters.html">
       <p>
        Comprehensive Profiling of Diverse Genetic Reporters with Application to Whole-Cell and Cell-Free Biosensors
       </p>
      </a>
      <p>
       Lopreside, A., Wan, X., Michelini, E., Roda, A. and Wang, B., in 2019
      </p>
      <p>
       <strong>
        Reporters
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Voigt Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Chen,Y2013.html">
       <p>
        Characterization of 582 natural and synthetic terminators and quantification of their design constraints
       </p>
      </a>
      <p>
       Chen, Y., Liu, P., Nielsen, A., Brophy, J., Clancy, K., Peterson, T. and Voigt, C., in 2013
      </p>
      <p>
       <strong>
        28 Terminators
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Stanton,B2013.html">
       <p>
        Genomic mining of prokaryotic repressors for orthogonal logic gates
       </p>
      </a>
      <p>
       Stanton, B., Nielsen, A., Tamsir, A., Clancy, K., Peterson, T. and Voigt, C., in 2013
      </p>
      <p>
       <strong>
        17 Regulated Promoters
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Rhodius,V2013.html">
       <p>
        Design of orthogonal genetic switches based on a crosstalk map of σs, anti‐σs, and promoters
       </p>
      </a>
      <p>
       Rhodius, V., Segall‐Shapiro, T., Sharon, B., Ghodasara, A., Orlova, E., Tabakh, H., Burkhardt, D., Clancy, K., Peterson, T., Gross, C. and Voigt, C., in 2013
      </p>
      <p>
       <strong>
        20 Regulated Promoters
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Nielsen,A2016.html">
       <p>
        Genetic circuit design automation
       </p>
      </a>
      <p>
       Nielsen, A., Der, B., Shin, J., Vaidyanathan, P., Paralanov, V., Strychalski, E., Ross, D., Densmore, D. and Voigt, C., in 2016
      </p>
      <p>
       <strong>
        16 Insulators
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Fernandez-Rodriguez,J2016.html">
       <p>
        Post-translational control of genetic circuits using Potyvirus proteases
       </p>
      </a>
      <p>
       Fernandez-Rodriguez, J. and Voigt, C., in 2016
      </p>
      <p>
       <strong>
        3 Cleavage Tags
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Meyer,A2018.html">
       <p>
        Escherichia coli “Marionette” strains with 12 highly optimized small-molecule sensors
       </p>
      </a>
      <p>
       Meyer, A., Segall-Shapiro, T., Glassey, E., Zhang, J. and Voigt, C., in 2018
      </p>
      <p>
       <strong>
        12 Regulated Promoters
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Yang,L2014.html">
       <p>
        Permanent Genetic Memory With &gt;1-byte Capacity
       </p>
      </a>
      <p>
       Yang, L., Nielsen, A., Fernandez-Rodriguez, J., McClune, C., Laub, M., Lu, T. and Voigt, C., in 2014
      </p>
      <p>
       <strong>
        13 Recombinases
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Park,Y2020.html">
       <p>
        Precision design of stable genetic circuits carried in highly‐insulated E. coli genomic landing pads
       </p>
      </a>
      <p>
       Park, Y., Espah Borujeni, A., Gorochowski, T., Shin, J. and Voigt, C., in 2020
      </p>
      <p>
       <strong>
        6 Terminators
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Salis Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Reis,A2019.html">
       <p>
        Simultaneous repression of multiple bacterial genes using nonrepetitive extra-long sgRNA arrays
       </p>
      </a>
      <p>
       Reis, A., Halper, S., Vezeau, G., Cetnar, D., Hossain, A., Clauer, P. and Salis, H., in 2019
      </p>
      <p>
       <strong>
        34 CRISPR Systems
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Endy Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Mutalik,V2013.html">
       <p>
        Precise and reliable gene expression via standard transcription and translation initiation elements
       </p>
      </a>
      <p>
       Mutalik, V., Guimaraes, J., Cambray, G., Lam, C., Christoffersen, M., Mai, Q., Tran, A., Paull, M., Keasling, J., Arkin, A. and Endy, D., in 2013
      </p>
      <p>
       <strong>
        1 Insulator
        <br/>
        70 Ribosome Binding Sites
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Cambray,G2013.html">
       <p>
        Measurement and modeling of intrinsic transcription terminators
       </p>
      </a>
      <p>
       Cambray, G., Guimaraes, J., Mutalik, V., Lam, C., Mai, Q., Thimmaiah, T., Carothers, J., Arkin, A. and Endy, D., in 2013
      </p>
      <p>
       <strong>
        16 Terminators
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Bonnet,J2013.html">
       <p>
        Amplifying Genetic Logic Gates
       </p>
      </a>
      <p>
       Bonnet, J., Yin, P., Ortiz, M., Subsoontorn, P. and Endy, D., in 2013
      </p>
      <p>
       <strong>
        2 Recombinases
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Sauer and Baker Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Wang,K2008.html">
       <p>
        Tuning the Strength of a Bacterial N-end Rule Degradation Signal
       </p>
      </a>
      <p>
       Wang, K., Oakes, E., Sauer, R. and Baker, T., in 2008
      </p>
      <p>
       <strong>
        4 Degradation Tags
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Davis,J2010.html">
       <p>
        Design, construction and characterization of a set of insulated bacterial promoters
       </p>
      </a>
      <p>
       Davis, J., Rubin, A. and Sauer, R., in 2010
      </p>
      <p>
       <strong>
        10 Constitutive Promoters
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Tabor Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_TaborLab.html">
       Tabor Lab publications
      </a>
     </li>
    </ul>
    <h2>
     Freemont Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Moore,S2016.html">
       <p>
        EcoFlex: A Multifunctional MoClo Kit for E. coli Synthetic Biology
       </p>
      </a>
      <p>
       Moore, S., Lai, H., Kelwick, R., Chee, S., Bell, D., Polizzi, K. and Freemont, P., in 2016
      </p>
      <p>
       <strong>
        10 Constitutive Promoters
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Lucks Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Chappell,J2017.html">
       <p>
        Computational design of small transcription activating RNAs for versatile and dynamic gene regulation
       </p>
      </a>
      <p>
       Chappell, J., Westbrook, A., Verosloff, M. and Lucks, J., in 2017
      </p>
      <p>
       <strong>
        15 Riboregulators
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Green Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Kim,J2019.html">
       <p>
        De novo-designed translation-repressing riboregulators for multi-input cellular logic
       </p>
      </a>
      <p>
       Kim, J., Zhou, Y., Carlson, P., Teichmann, M., Chaudhary, S., Simmel, F., Silver, P., Collins, J., Lucks, J., Yin, P. and Green, A., in 2019
      </p>
      <p>
       <strong>
        27 Riboregulators
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Lou Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Zong,Y2017.html">
       <p>
        Insulated transcriptional elements enable precise design of genetic circuits
       </p>
      </a>
      <p>
       Zong, Y., Zhang, H., Lyu, C., Ji, X., Hou, J., Guo, X., Ouyang, Q. and Lou, C., in 2017
      </p>
      <p>
       <strong>
        8 Regulated Promoters
       </strong>
      </p>
     </li>
     <li>
      <a href="../tables/pub_Du,P2020.html">
       <p>
        De novo design of an intercellular signaling toolbox for multi-channel cell–cell communication and biological computation
       </p>
      </a>
      <p>
       Du, P., Zhao, H., Zhang, H., Wang, R., Huang, J., Tian, Y., Luo, X., Luo, X., Wang, M., Xiang, Y., Qian, L., Chen, Y., Tao, Y. and Lou, C., in 2020
      </p>
      <p>
       <strong>
        8 Regulated Promoters
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Müller Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Jahn,M2016.html">
       <p>
        Copy number variability of expression plasmids determined by cell sorting and Droplet Digital PCR
       </p>
      </a>
      <p>
       Jahn, M., Vorpahl, C., Hübschmann, T., Harms, H. and Müller, S., in 2016
      </p>
      <p>
       <strong>
        7 Origins of Replication
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Hasty Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Scott,S2016.html">
       <p>
        Quorum Sensing Communication Modules for Microbial Consortia
       </p>
      </a>
      <p>
       Scott, S. and Hasty, J., in 2016
      </p>
      <p>
       <strong>
        4 Regulated Promoters
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Polizzi Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Kylilis,N2018.html">
       <p>
        Tools for engineering coordinated system behaviour in synthetic microbial consortia
       </p>
      </a>
      <p>
       Kylilis, N., Tuza, Z., Stan, G. and Polizzi, K., in 2018
      </p>
      <p>
       <strong>
        6 Regulated Promoters
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Church Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Esvelt,K2013.html">
       <p>
        Orthogonal Cas9 proteins for RNrnA-guided gene regulation and editing
       </p>
      </a>
      <p>
       Esvelt, K., Mali, P., Braff, J., Moosburner, M., Yaung, S. and Church, G., in 2013
      </p>
      <p>
       <strong>
        4 CRISPR Systems
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Forster Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/type_reporters.html">
       <p>
        Engineering a palette of eukaryotic chromoproteins for bacterial synthetic biology
       </p>
      </a>
      <p>
       Liljeruhm, J., Funk, S.K., Tietscher, S., Edlund, A.D., Jamal, S., Wistrand-Yuen, P., Dyrhage, K., Gynnå, A., Ivermark, K., Lövgren, J., Törnblom, V., Virtanen, A., Lundin, E.R., Wistrand-Yuen, E. and Forster, A.C., in 2018
      </p>
      <p>
       <strong>
        Reporters
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Collins Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Cameron,D2014.html">
       <p>
        Tunable protein degradation in bacteria
       </p>
      </a>
      <p>
       Cameron, D. and Collins, J., in 2014
      </p>
      <p>
       <strong>
        7 Degradation Tags
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Molin Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Andersen,J1998.html">
       <p>
        New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria
       </p>
      </a>
      <p>
       Andersen, J., Sternberg, C., Poulsen, L., Bjørn, S., Givskov, M. and Molin, S., in 1998
      </p>
      <p>
       <strong>
        4 Degradation Tags
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Möglich Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Ohlendorf,R2012.html">
       <p>
        From Dusk till Dawn: One-Plasmid Systems for Light-Regulated Gene Expression
       </p>
      </a>
      <p>
       Ohlendorf, R., Vidavski, R., Eldar, A., Moffat, K. and Möglich, A., in 2012
      </p>
      <p>
       <strong>
        1 Regulated Promoter
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Yang Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Li,X2020.html">
       <p>
        A single-component light sensor system allows highly tunable and direct activation of gene expression in bacterial cells
       </p>
      </a>
      <p>
       Li, X., Zhang, C., Xu, X., Miao, J., Yao, J., Liu, R., Zhao, Y., Chen, X. and Yang, Y., in 2020
      </p>
      <p>
       <strong>
        1 Regulated Promoter
       </strong>
      </p>
     </li>
    </ul>
    <h2>
     Wilson Lab
    </h2>
    <ul>
     <li>
      <a href="../tables/pub_Groseclose,T2020.html">
       <p>
        Engineered systems of inducible anti-repressors for the next generation of biological programming
       </p>
      </a>
      <p>
       Groseclose, T., Rondon, R., Herde, Z., Aldrete, C. and Wilson, C., in 2020
      </p>
      <p>
       <strong>
        14 Regulated Promoters
       </strong>
      </p>
     </li>
    </ul>
   </div>
  </div>
 </body>
</html>