---
layout: page
title: 发表论文 
show_sidebar: false
# hero_image: https://www.restoringautoimmunehealth.com/wp-content/uploads/2019/01/DNA-image-1080x640.jpg
hero_image: /assets/dnabackground.jpg
toc: true
---


## 代表论文

* Liu Y, Pinto F, Wan X, Z Yang, Peng S, Li M, J Cooper, Xie Z, French C and **Wang B\***, “Reprogrammed tracrRNAs enable repurposing of RNAs as crRNAs and sequence-specific RNA biosensors”, _**Nature Communications**_, 2022, 13, 1937  [doi](https://doi.org/10.1038/s41467-022-29604-x)  [pdf](https://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Liu2022NatCommun_Reprogrammed%20tracrRNAs%20enable%20repurposing%20of%20RNAs%20as%20crRNAs%20and%20sequence-specific%20RNA%20biosensors.pdf)  (Reported by [Nature Research Bioengineering Blog](https://bioengineeringcommunity.nature.com/posts/repurposing-rnas-to-be-crispr-rnas-starting-with-engineering#share), [Regensis](https://mp.weixin.qq.com/s/yn6CjiMARo_ABv_mnI7SVg), [BioArtMED](http://mp.weixin.qq.com/s?__biz=MzU4OTY0NzE1NQ==&mid=2247512419&idx=3&sn=f6477361570e0915b4cc040f7f694e1f&chksm=fdc8ad3fcabf2429b2432cc8e491a254502aab6bddfc35283c57c8dd8298f311cd1d4a95e2d1&mpshare=1&scene=1&srcid=0430JQOiCHVdRAFCAPvrtUPh&sharer_sharetime=1651313592565&sharer_shareid=6ddaed0a58ce0a547d64e85841dd92aa#rd), [X-MOL](https://mp.weixin.qq.com/s/HTSpzmZgC2KoyFb6QBz8sA), [biosyn-met](https://mp.weixin.qq.com/s/CMCOrR195KfgDQ36KHUb9w) and [LanHanBiotechnology](https://mp.weixin.qq.com/s/s_WKqSnkbmx16Rt84mZJzQ) among others)  
<br>   

*   Ho T, Shao Y, Lu Z, Savilahti H, Menolascina F, Wang L, Dalchau N and **Wang B**\*, “A systematic approach to inserting split inteins for Boolean logic gate engineering and basal activity reduction”, _**Nature Communications**_, 2021, 12, 2200  [doi](https://doi.org/10.1038/s41467-021-22404-9)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Ho2021NatCommun_A%20systematic%20approach%20to%20inserting%20split%20inteins%20for%20Boolean%20logic%20gate%20engineering%20and%20basal%20activity%20reduction.pdf)  (Reported by  [PhysOrg](http://phys.org/news/2021-04-proteins-cells-molecular.html), [Edinburgh BioSciences News](https://www.ed.ac.uk/biology/news-events/news-2021/programming-proteins-to-turn-cells-into-molecular), [UoE SynthSys News](http://www.ed.ac.uk/biology/synthsys/news-and-events/news/biocomputation-made-easy-by-a-new-method-in-cracki) and [BioArt](http://mp.weixin.qq.com/s/fGFWzRHA01PcKl-KcseSLA) among others)  
<br>  

*   Wan X, Pinto F, Yu L and **Wang B**\*, “Synthetic protein-binding DNA sponge as a tool to tune gene expression and mitigate protein toxicity”, **_Nature Communications_**, 2020, 11, 5961  [doi](https://doi.org/10.1038/s41467-020-19552-9)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wan2020NatCommun_Synthetic%20protein-binding%20DNA%20sponge%20as%20a%20tool%20to%20tune%20gene%20expression%20and%20mitigate%20protein%20toxicity.pdf)  (Reported by [Nature Research Bioengineering Blog](https://bioengineeringcommunity.nature.com/posts/synthetic-dna-sponge), [BioArt](https://wemp.app/posts/7571fc12-d521-491b-86ad-57b0ec461681), [Zhihu-Regensis](https://zhuanlan.zhihu.com/p/314520966) and [UoE SynthSys News](https://www.ed.ac.uk/biology/synthsys/news-and-events/news-2020/dna-sponges-new-tool-for-fine-tuning-cell-engine) among others)  
<br>  

*   Pinto F, Thornton E and **Wang B**\*, “An expanded library of orthogonal split inteins enables modular multi-peptide assemblies”, **_Nature Communications_**, 2020, 11, 1529  [doi](https://doi.org/10.1038/s41467-020-15272-2)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Pinto2020NatCommun_An%20expanded%20library%20of%20orthogonal%20split%20inteins%20enables%20modular%20multi-peptide%20assemblies.pdf)  (Reported by [Edinburgh BioSciences News](https://www.ed.ac.uk/biology/news-events/news-2020/harnessing-nature-s-protein-glue-could-aid-medicin), [Nature Research Bioengineering Blog](https://bioengineeringcommunity.nature.com/posts/61769-orthogonal-split-inteins-for-multi-peptide-assemblies) and [BioArtReports](http://mp.weixin.qq.com/s?__biz=MzU4OTY0NzE1NQ==&mid=2247488793&idx=3&sn=2b9345d3b9b1adc9b44196cd40a838ed&chksm=fdcb0945cabc8053d689314e5dc1a0366cb7a07514a5f3bbf730656ed91efaee0f7bae3234cd#rd) among others)  
<br>  
    

*   Wan X, Volpetti F, Petrova E, French C, Maerkel S and **Wang B**\*, "Cascaded amplifying circuits enable ultrasensitive cellular sensors for toxic metals", _**Nature Chemical Biology**_, 2019, 15(5):540–548  [doi](https://doi.org/10.1038/s41589-019-0244-3)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wan2019NatChemBiol_Cascaded%20amplifying%20circuits%20enable%20ultrasensitive%20cellular%20sensors%20for%20toxic%20metals.pdf)  (Reported by [Yahoo News](https://uk.news.yahoo.com/smartphone-device-used-detect-poisonous-143315760.html?soc_src=social-sh&soc_trk=ma), [E&T Magazine](https://eandt.theiet.org/content/articles/2019/03/smartphone-biosensor-detects-arsenic-in-drinking-water/), [PhysOrg](https://phys.org/news/2019-03-smartphone-poisoned-millions.html), [Science Daily](https://www.sciencedaily.com/releases/2019/03/190326132650.htm), [Microbiologist](https://sfam.org.uk/resources/microbiologist-2019-september.html), [BioArt](https://wemp.app/posts/aae972fb-c5fd-48f4-83e9-f0714061f742), [Edinburgh Friends](https://www.ed.ac.uk/edinburgh-friends/supplements/how-smartphone-become-lifesaver), [Edinburgh University News](https://www.ed.ac.uk/news/2019/smartphone-test-spots-poisoned-water-risk) and [UoE BioSciences News](https://www.ed.ac.uk/biology/news-events/news-2019/smartphone-based-biosensor-to-tackle-arsenic-poiso) among others)  
<br>  
    

*   Liu Y, Wan X and **Wang B**\*, "Engineered CRISPRa enables programmable eukaryote-like gene activation in bacteria", _**Nature Communications**_, 2019, 10, 3693  [doi](https://doi.org/10.1038/s41467-019-11479-0)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Liu2019NatCommun_Engineered%20CRISPRa%20enables%20programmable%20eukaryote-like%20gene%20activation%20in%20bacteria.pdf)  (Reported by [The National](http://thenational.scot/news/17861583.scottish-crispr-progress-allows-gene-activations-industry/?ref=rss), [The National Tribune](https://www.nationaltribune.com.au/hi-tech-bacteria-gene-tool-could-prove-productive/),  [Drug Target Review](https://www.drugtargetreview.com/news/48222/novel-bacteria-gene-activation-method-developed/), [PhysOrg](https://phys.org/news/2019-08-hi-tech-bacteria-gene-tool-productive.html), [Science Daily](https://www.sciencedaily.com/releases/2019/08/190826110406.htm), [BioArt](https://wemp.app/posts/496755f4-88f7-423b-b763-60b2555b6818), [Zhihu-Regensis](http://zhuanlan.zhihu.com/p/80044731), [Edinburgh University News](https://www.ed.ac.uk/news/2019/hi-tech-bacteria-gene-tool-could-prove-productive) and [UoE SynthSys News](http://www.synthsys.ed.ac.uk/news/hi-tech-bacteria-gene-tool-revealed) among others)  
<br>  
    

*   **Wang B**, Kitney R, Joly N and Buck M, “Engineering modular and orthogonal genetic logic gates for robust digital-like synthetic biology”, **_Nature Communications_**, 2011, 2:508  [doi](http://www.nature.com/ncomms/journal/v2/n10/full/ncomms1516.html)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2011NatureComms_biological_gates.pdf)  (Reported by _[Financial Times](http://www.ft.com/cms/s/2/25a1f6d4-ff5e-11e0-aa11-00144feabdc0.html#axzz1cjfiNUDt)_, [EPSRC](http://www.epsrc.ac.uk/newsevents/news/computermaygrow/), [European Commission](https://cordis.europa.eu/news/rcn/33946/en), _[Electronics Weekly](http://www.electronicsweekly.com/Articles/18/10/2011/52071/logic-gates-from-bacteria-and-dna.htm)_, _[Science Daily](http://www.sciencedaily.com/releases/2011/10/111018111929.htm)_, [UK Synthetic Biology Roadmap](https://connect.innovateuk.org/documents/2826135/3815409/Synthetic+Biology+Roadmap+-+Report.pdf/fa8a1e8e-cbf4-4464-87ce-b3b033f04eaa) and [Imperial College News](http://www3.imperial.ac.uk/newsandeventspggrp/imperialcollege/newssummary/news_18-10-2011-16-7-29) among others)  
<br>  
    
*   **Wang B**\*, Barahona M and Buck M, “Engineering modular and tunable genetic amplifiers for scaling transcriptional signals in cascaded gene networks”, **_Nucleic Acids Research_**, 2014, 42(14): 9484-9492  [doi](http://dx.doi.org/10.1093/nar/gku593)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Nucl.%20Acids%20Res.-2014-Wang_Engineering%20modular%20and%20tunable%20genetic%20amplifiers%20for%20scaling%20transcriptional%20signals%20in%20cascaded%20gene%20networks.pdf)  (Featured in [Imperial College Science News](http://www3.imperial.ac.uk/newsandeventspggrp/imperialcollege/newssummary/news_24-7-2014-17-21-47))  
<br>  
    
*   **Wang B**\*, Barahona M and Buck M, “Amplification of small molecule-inducible gene expression via tuning of intracellular receptor densities”, **_Nucleic Acids Research_**, 2015, 43(3): 1955-1964  [doi](http://dx.doi.org/10.1093/nar/gku1388)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2015NuclAcidsRes_Amplifying%20gene%20expression%20.pdf)  
<br>  
    
    
*   **Wang B**\*, Barahona M and Buck M, “A modular cell-based biosensor using engineered genetic logic circuits to detect and integrate multiple environmental signals”, **_Biosensors and Bioelectronics_**, 2013, 40(1): 368-376  [doi](http://dx.doi.org/10.1016/j.bios.2012.08.011)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2013BiosensorsBioelectronics_biosensor.pdf)  
<br>  
    

  

## 期刊论文

*  Fa-arun J, Huan Y, E Darmon and**Wang B\***, “Tail-engineered phage P2 enables delivery of antimicrobial phagemids into multiple gut pathogens”, _**ACS Synthetic Biology**_, 2023, 12, 2, 596–607  [doi](https://pubs.acs.org/doi/10.1021/acssynbio.2c00615)  
<br>  

*  Huan Y#, Torraca V#, Brown R#, Fa-arun J, Miles S, Oyarzun D, Mostowy S* and **Wang B\***, “P1 bacteriophage-enabled delivery of CRISPR-Cas9 antimicrobial activity against Shigella flexneri”, _**ACS Synthetic Biology**_, 2023, 12, 3, 709–721  [doi](https://pubs.acs.org/doi/10.1021/acssynbio.2c00465)  
<br>  

*  Huan Y, Brown R and **Wang B\***, “An adenine/thymidine-rich region is integral to RepL-mediated DNA replication”, _**Frontiers in Microbiology**_, 2023, 14, 1095671  [doi](https://doi.org/10.3389/fmicb.2023.1095671)  
<br>  

*   Liu Y, Pinto F, Wan X, Z Yang, Peng S, Li M, J Cooper, Xie Z, French C and **Wang B\***, “Reprogrammed tracrRNAs enable repurposing of RNAs as crRNAs and sequence-specific RNA biosensors”, _**Nature Communications**_, 2022, 13, 1937  [doi](https://doi.org/10.1038/s41467-022-29604-x)  [pdf](https://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Liu2022NatCommun_Reprogrammed%20tracrRNAs%20enable%20repurposing%20of%20RNAs%20as%20crRNAs%20and%20sequence-specific%20RNA%20biosensors.pdf)  (Reported by [Nature Research Bioengineering Blog](https://bioengineeringcommunity.nature.com/posts/repurposing-rnas-to-be-crispr-rnas-starting-with-engineering#share), [Regensis](https://mp.weixin.qq.com/s/yn6CjiMARo_ABv_mnI7SVg), [BioArtMED](http://mp.weixin.qq.com/s?__biz=MzU4OTY0NzE1NQ==&mid=2247512419&idx=3&sn=f6477361570e0915b4cc040f7f694e1f&chksm=fdc8ad3fcabf2429b2432cc8e491a254502aab6bddfc35283c57c8dd8298f311cd1d4a95e2d1&mpshare=1&scene=1&srcid=0430JQOiCHVdRAFCAPvrtUPh&sharer_sharetime=1651313592565&sharer_shareid=6ddaed0a58ce0a547d64e85841dd92aa#rd), [X-MOL](https://mp.weixin.qq.com/s/HTSpzmZgC2KoyFb6QBz8sA), [biosyn-met](https://mp.weixin.qq.com/s/CMCOrR195KfgDQ36KHUb9w) and [LanHanBiotechnology](https://mp.weixin.qq.com/s/s_WKqSnkbmx16Rt84mZJzQ) among others)  
<br>  

*   Huan Y, Fa-arun J and **Wang B**\*, “The role of O-antigen in P1 transduction of _Shigella flexneri_ and _Escherichia coli_ with its alternative S' tail fibre”, **_Journal of Molecular Biology_**, 2022, 434(21), 167829  [doi](https://doi.org/10.1016/j.jmb.2022.167829)  
<br>  

*   Pan Y, Mao K, Hui Q, **Wang B**, Cooper J and Zhugen Yang\*, “Paper-based devices for rapid diagnosis and wastewater surveillance”, **_Trends in Analytical Chemistry_**, 2022, 157, 116760  [doi](https://doi.org/10.1016/j.trac.2022.116760)  
<br>  

*   Chen SY, Zhang Y, Li R, **Wang B**\* and Ye BC\*, “_De novo_ design of the ArsR regulated _Pars_ promoter enables a highly sensitive whole-cell biosensor for arsenic contamination”, _**Analytical Chemistry**_, 2022, 94(20), 7210–7218  [doi](https://doi.org/10.1021/acs.analchem.2c00055)  [pdf](https://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Chen2022AnalChem_De%20Novo%20Design%20of%20the%20ArsR%20Regulated%20Pars%20Promoter%20Enables%20a%20Highly%20Sensitive%20Whole-Cell%20Biosensor.pdf)  
<br>  
    

*   Li S, Xiao J, Sun T, Yu F, Zhang K, Feng Y, Xu C, **Wang B** and Cheng L, “Synthetic microbial consortia with programmable ecological interactions”, _**Methods in Ecology and Evolution**_, 2022, 13(7), 1608-1621  [doi](https://doi.org/10.1111/2041-210X.13894)  [pdf](https://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Li2022Methods%20Ecol%20Evol_Synthetic%20microbial%20consortia%20with%20programmable%20ecological%20interactions.pdf)  
<br>  
    

*   **Saltepe B**, Wang L and **Wang B**\*, “Synthetic biology enables field-deployable biosensors for water contaminants”, _**Trends in Analytical Chemistry**_, 2022, 146, 116507  [doi](https://doi.org/10.1016/j.trac.2021.116507)  [pdf](https://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Saltepe2022TrAC_Synthetic%20biology%20enables%20field-deployable%20biosensors%20for%20water%20contaminants.pdf)  
<br>  
    

*   Belkin S\* and **Wang B**\*, “Sense and sensibility: of synthetic biology and the redesign of bioreporter circuits”, _**Microbial Biotechnology**_, 2022, 15(1), 103-106  [doi](https://doi.org/10.1111/1751-7915.13955)  [pdf](https://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Belkin2022MicrobialBiotech_Sense%20and%20sensibility%20of%20synthetic%20biology%20and%20the%20redesign%20of%20bioreporter%20circuits.pdf)  
<br>  
    



*   Ho T, Shao Y, Lu Z, Savilahti H, Menolascina F, Wang L, Dalchau N and **Wang B**\*, “A systematic approach to inserting split inteins for Boolean logic gate engineering and basal activity reduction”, _**Nature Communications**_, 2021, 12, 2200  [doi](https://doi.org/10.1038/s41467-021-22404-9)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Ho2021NatCommun_A%20systematic%20approach%20to%20inserting%20split%20inteins%20for%20Boolean%20logic%20gate%20engineering%20and%20basal%20activity%20reduction.pdf)  (Reported by  [PhysOrg](http://phys.org/news/2021-04-proteins-cells-molecular.html), [Edinburgh BioSciences News](https://www.ed.ac.uk/biology/news-events/news-2021/programming-proteins-to-turn-cells-into-molecular), [UoE SynthSys News](http://www.ed.ac.uk/biology/synthsys/news-and-events/news/biocomputation-made-easy-by-a-new-method-in-cracki) and [BioArt](http://mp.weixin.qq.com/s/fGFWzRHA01PcKl-KcseSLA) among others)  
<br>  
    

*   Lopreside A, Montali L, **Wang B**, Tassoni A, Ferri M, Calabretta M and Michelini E, “Orthogonal paper biosensor for mercury(II) contamination combining bioluminescence and colorimetric smartphone detection”, **_Biosensors and Bioelectronics_**, 2021, 194, 113569  [doi](https://doi.org/10.1016/j.bios.2021.113569)  [pdf](https://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Lopreside2021BB_Orthogonal%20paper%20biosensor%20for%20mercury%28II%29%20combining%20bioluminescence%20and%20colorimetric%20smartphone%20detection.pdf)  
<br>  
    

*   Chen JX, **Wang B**, Thompson I, Huang EW, “Rational design and characterisation of nitric oxide biosensors in _E. coli_ Nissle 1917 and mini SimCells”, **_ACS Synthetic Biology_**, 2021, 10(10), 2566–2578.  [doi](https://doi.org/10.1021/acssynbio.1c00223)  [pdf](https://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Chen2021acssynbio_Rational%20Design%20and%20Characterization%20of%20Nitric%20Oxide%20Biosensors%20in%20E.%20coli%20Nissle%201917%20and%20Mini%20SimCells.pdf)  
<br>  
    

*   **Wan X**, **Saltepe B**, Yu L and **Wang B**\*, “Programming living sensors for environment, health and biomanufacturing”, _**Microbial Biotechnology**_, 2021, 14(6): 2334-2342.  [doi](https://doi.org/10.1111/1751-7915.13820)  [pdf](https://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wan2021MicrobialBiotech_Programming%20living%20sensors%20for%20environment%2C%20health%20and%20biomanufacturing.pdf)  
<br>  
    


*   Liu Y, Pinto F, Wan X, Peng S, Li M, Xie Z, French C and **Wang B**\*, “Reprogrammed tracrRNAs enable repurposing RNAs as crRNAs and detecting RNAs”, **_bioRxiv_**, 2021  [doi](https://doi.org/10.1101/2021.05.24.445356)  
<br>  
    

*   Yang S, Yang X, **Wang B** and Wang L, “Design and applications of fluorogenic nucleic acid paper-based biosensors”, _**Progress in Chemistry**_, 2021, 33(12), 2309-2315.  [doi](https://manu56.magtech.com.cn/progchem/EN/10.7536/PC201129)  [pdf](http://sbsweb2.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Yang2021%E5%8C%96%E5%AD%A6%E8%BF%9B%E5%B1%95_%E5%9F%BA%E4%BA%8E%E6%A0%B8%E9%85%B8%E7%9A%84%E7%BA%B8%E5%9F%BA%E8%8D%A7%E5%85%89%E7%94%9F%E7%89%A9%E4%BC%A0%E6%84%9F%E5%99%A8%E7%9A%84%E8%AE%BE%E8%AE%A1%E5%8F%8A%E5%BA%94%E7%94%A8.pdf)  
<br>  
    

*   Gale G, **Wang B** and McCormick A, “Evaluation and comparison of the efficiency of transcription terminators in different cyanobacterial species", _**Frontiers in Microbiology**_, 2021, 11:624011  [doi](https://doi.org/10.3389/fmicb.2020.624011)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Gale2021FrontMicrobiol_Evaluation%20and%20Comparison%20of%20the%20Efficiency%20of%20Transcription%20Terminators%20in%20Different%20Cyanobacterial%20Species.pdf)  
<br>  
    

*   Wan X, Pinto F, Yu L and **Wang B**\*, “Synthetic protein-binding DNA sponge as a tool to tune gene expression and mitigate protein toxicity”, **_Nature Communications_**, 2020, 11, 5961  [doi](https://doi.org/10.1038/s41467-020-19552-9)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wan2020NatCommun_Synthetic%20protein-binding%20DNA%20sponge%20as%20a%20tool%20to%20tune%20gene%20expression%20and%20mitigate%20protein%20toxicity.pdf)  (Reported by [Nature Research Bioengineering Blog](https://bioengineeringcommunity.nature.com/posts/synthetic-dna-sponge), [BioArt](https://wemp.app/posts/7571fc12-d521-491b-86ad-57b0ec461681), [Zhihu-Regensis](https://zhuanlan.zhihu.com/p/314520966) and [UoE SynthSys News](https://www.ed.ac.uk/biology/synthsys/news-and-events/news-2020/dna-sponges-new-tool-for-fine-tuning-cell-engine) among others)  
<br>  
    

*   Pinto F, Thornton E and **Wang B**\*, “An expanded library of orthogonal split inteins enables modular multi-peptide assemblies”, **_Nature Communications_**, 2020, 11, 1529  [doi](https://doi.org/10.1038/s41467-020-15272-2)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Pinto2020NatCommun_An%20expanded%20library%20of%20orthogonal%20split%20inteins%20enables%20modular%20multi-peptide%20assemblies.pdf)  (Reported by [Edinburgh BioSciences News](https://www.ed.ac.uk/biology/news-events/news-2020/harnessing-nature-s-protein-glue-could-aid-medicin), [Nature Research Bioengineering Blog](https://bioengineeringcommunity.nature.com/posts/61769-orthogonal-split-inteins-for-multi-peptide-assemblies) and [BioArtReports](http://mp.weixin.qq.com/s?__biz=MzU4OTY0NzE1NQ==&mid=2247488793&idx=3&sn=2b9345d3b9b1adc9b44196cd40a838ed&chksm=fdcb0945cabc8053d689314e5dc1a0366cb7a07514a5f3bbf730656ed91efaee0f7bae3234cd#rd) among others)  
<br>  
    

*   Liu Y and **Wang B**\*, “A novel eukaryote-like CRISPR activation tool in bacteria: features and capabilities”, _**BioEssays**_, 2020, 42(6), 1900252  [doi](https://doi.org/10.1002/bies.201900252)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Liu2020BioEssays_A%20novel%20eukaryote%E2%80%90like%20CRISPR%20activation%20tool%20in%20bacteria%20-features%20and%20capabilities.pdf)  (Featured as Journal [Front Cover Article](https://doi.org/10.1002/bies.202070061))  
<br>  
    

*   Hicks M, Bachmann T and **Wang B**\*, “Synthetic biology enables programmable cell-based biosensors”, _**ChemPhysChem**_, 2020, 21, 132–144  [doi](https://doi.org/10.1002/cphc.201900739)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Hicks2020ChemPhyChem_Synthetic%20biology%20enables%20cell-based%20biosensors.pdf)  (Featured as Journal [Front Cover Article](https://doi.org/10.1002/cphc.201901192); [_ChemPhysChem_ Top Downloaded Paper](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Top%20Downloaded%20Paper%20ChemPhysChem%202018-2019%20landscape.pdf))  
<br>  
    

*   Liu Y, Wan X and **Wang B**\*, "Engineered CRISPRa enables programmable eukaryote-like gene activation in bacteria", _**Nature Communications**_, 2019, 10, 3693  [doi](https://doi.org/10.1038/s41467-019-11479-0)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Liu2019NatCommun_Engineered%20CRISPRa%20enables%20programmable%20eukaryote-like%20gene%20activation%20in%20bacteria.pdf)  (Reported by [The National](http://thenational.scot/news/17861583.scottish-crispr-progress-allows-gene-activations-industry/?ref=rss), [The National Tribune](https://www.nationaltribune.com.au/hi-tech-bacteria-gene-tool-could-prove-productive/),  [Drug Target Review](https://www.drugtargetreview.com/news/48222/novel-bacteria-gene-activation-method-developed/), [PhysOrg](https://phys.org/news/2019-08-hi-tech-bacteria-gene-tool-productive.html), [Science Daily](https://www.sciencedaily.com/releases/2019/08/190826110406.htm), [BioArt](https://wemp.app/posts/496755f4-88f7-423b-b763-60b2555b6818), [Zhihu-Regensis](http://zhuanlan.zhihu.com/p/80044731), [Edinburgh University News](https://www.ed.ac.uk/news/2019/hi-tech-bacteria-gene-tool-could-prove-productive) and [UoE SynthSys News](http://www.synthsys.ed.ac.uk/news/hi-tech-bacteria-gene-tool-revealed) among others)  
<br>  
    

*   Wan X, Volpetti F, Petrova E, French C, Maerkel S and **Wang B**\*, "Cascaded amplifying circuits enable ultrasensitive cellular sensors for toxic metals", _**Nature Chemical Biology**_, 2019, 15(5):540–548  [doi](https://doi.org/10.1038/s41589-019-0244-3)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wan2019NatChemBiol_Cascaded%20amplifying%20circuits%20enable%20ultrasensitive%20cellular%20sensors%20for%20toxic%20metals.pdf)  (Reported by [Yahoo News](https://uk.news.yahoo.com/smartphone-device-used-detect-poisonous-143315760.html?soc_src=social-sh&soc_trk=ma), [E&T Magazine](https://eandt.theiet.org/content/articles/2019/03/smartphone-biosensor-detects-arsenic-in-drinking-water/), [PhysOrg](https://phys.org/news/2019-03-smartphone-poisoned-millions.html), [Science Daily](https://www.sciencedaily.com/releases/2019/03/190326132650.htm), [Microbiologist](https://sfam.org.uk/resources/microbiologist-2019-september.html), [BioArt](https://wemp.app/posts/aae972fb-c5fd-48f4-83e9-f0714061f742), [Edinburgh Friends](https://www.ed.ac.uk/edinburgh-friends/supplements/how-smartphone-become-lifesaver), [Edinburgh University News](https://www.ed.ac.uk/news/2019/smartphone-test-spots-poisoned-water-risk) and [UoE BioSciences News](https://www.ed.ac.uk/biology/news-events/news-2019/smartphone-based-biosensor-to-tackle-arsenic-poiso) among others)  
<br>  
    

*   Lopreside A, Wan X, Michelini E, Roda A and **Wang B**\*, “Comprehensive profiling of diverse genetic reporters with application to whole-cell and cell-free biosensors”, _**Analytical Chemistry**_, 2019, 91, 15284−15292  [doi](https://doi.org/10.1021/acs.analchem.9b04444)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Lopreside2019analchem_Comprehensive%20profiling%20of%20diverse%20genetic%20reporters%20with%20application%20to%20whole-cell%20and%20cell-free%20biosensors.pdf)  
<br>  
    

*   Gale G, Schiavon A, Mills L, **Wang B**, Lea-Smith D and McCormick A, “Emerging species and genome editing tools: future prospects in cyanobacterial synthetic biology”, _**Microorganisms**_, 2019, 7(10):409  [doi](https://doi.org/10.3390/microorganisms7100409)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Gale2019Microorganisms_Emerging%20Species%20and%20Genome%20Editing%20Tools-%20Future%20Prospects%20in%20Cyanobacterial%20Synthetic%20Biology.pdf)  
<br>  
    

*   Gale G, Schiavon A, Puzorjov A, **Wang B** and McCormick A, “Genetic modification of cyanobacteria by conjugation using the modular cloning CyanoGate toolkit”, _**Journal of Visualized Experiments**_, 2019, (152), e60451 [doi](https://doi.org/10.3791/60451)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Gale2019jove_Genetic%20modification%20of%20cyanobacteria%20by%20conjugation%20using%20the%20CyanoGate%20modular%20cloning%20toolkit.pdf)  
<br>  
    

*   Vasudevan R, Gale G, Schiavon A, Puzorjov A, Malin J, Gillespie M, Vavitsas K, Zulkower V, **Wang B,** Howe C, Lea-Smith D and McCormick A, "CyanoGate: A modular cloning suite for engineering cyanobacteria based on the plant MoClo syntax", _**Plant Physiology**_, 2019, 180(1):39-55  [doi](https://doi.org/10.1104/pp.18.01401)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Gale2019PlantPhysiology_CyanoGate%20-a%20modular%20cloning%20suite%20for%20engineering%20cyanobacteria%20based%20on%20the%20plant%20MoClo%20syntax.pdf)  
<br>  
    

*   Liu Q, Schumacher J, Wan X, Lou C and **Wang B**\*, “Orthogonality and burdens of heterologous AND gate gene circuits in _E. coli_”, **_ACS Synthetic Biology_**, 2018, 7(2):553-564  [doi](http://dx.doi.org/10.1021/acssynbio.7b00328)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Liu2017acssynbio_Orthogonality%20and%20burdens%20of%20heterologous%20AND%20gate%20gene%20circuits%20in%20E.%20coli.pdf)  (Featured in [Edinburgh SynthSys Centre News](http://www.synthsys.ed.ac.uk/news/fine-balance-between-stress-and-success-synthetic-circuit-design); among [Most Read Articles of _ACS Synthetic Biology_](https://pubs.acs.org/action/showMostReadArticles?topArticlesType=recent&journalCode=asbcd6))  
<br>  
    

*   Hou J, Zeng W, Zong Y, Miao C, **Wang B** and Lou C, “Engineering the ultrasensitive transcription factors by fusing a modular oligomerization domain”, **_ACS Synthetic Biology_**, 2018, 7(5):1188–1194  [doi](http://dx.doi.org/10.1021/acssynbio.7b00414)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Hou2017ACSSynbio_Engineering%20the%20ultrasensitive%20transcription%20factors%20by%20fusing%20a%20modular%20oligomerisation%20domain.pdf)  
<br>  
    

*   Xiang Y, Dalchau N and **Wang B**\*, "Scaling up genetic circuit design for cellular computing: advances and prospects", **_Natural Computing_**, 2018, 17(4):833-853  [doi](http://dx.doi.org/10.1007/s11047-018-9715-9)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Xiang2018NaturalComputing_Scaling%20up%20genetic%20circuit%20design%20for%20cellular%20computing-advances%20and%20prospects.pdf)  
<br>  
    

*   Bernard E and **Wang B**\*, "Synthetic cell-based sensors with programmed selectivity and sensitivity", _**Methods in Molecular Biology**_, 2017, 1572:349-363  [doi](http://dx.doi.org/10.1007/978-1-4939-6911-1_23)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Bernard2017MethodsMB_Synthetic%20Cell-based%20Bisoensors%20_Chapter23.pdf)  
<br>   
    

*   Brown R, Lengeling A and **Wang B**\*, “Phage engineering: how advances in molecular biology and synthetic biology are being utilized to enhance the therapeutic potential of bacteriophages”, **_Quantitative Biology_**, 2017, 5(1):42-54  [doi](http://dx.doi.org/10.1007/s40484-017-0094-5)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Brown2017QuanBiol_Phage%20engineering-how%20advances%20in%20molecular%20biology%20and%20synthetic%20biology%20are%20being%20utilized%20to%20enhance%20the%20therapeutic%20potential%20of%20bacteriophages.pdf)  
<br>  
    

*   Bradley R, Buck M\* and **Wang B**\*, “Recognizing and engineering digital-like logic gates and switches in gene regulatory networks”, **_Current Opinion in Microbiology_**, 2016, 33:74–82  [doi](http://dx.doi.org/10.1016/j.mib.2016.07.004)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Bradley2016CoM_Recognizing%20and%20engineering%20digital-like%20logic%20gates%20and%20switches%20in%20gene%20regulatory%20networks.pdf)  
<br>  
    

*   Bradley R, Buck M and **Wang B**\*, “Tools and principles for microbial gene circuit engineering”, **_Journal of Molecular Biology_**, 2016, 428(5): 862-888  [doi](http://dx.doi.org/10.1016/j.jmb.2015.10.004)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Bradley2015JMB_Tools%20for%20microbial%20gene%20circuit%20engineering.pdf)  
<br>  
    

*   Bradley R and **Wang B**\*, “Designer cell signal processing circuits for biotechnology”, **_New Biotechnology_**, 2015, 32(6): 635-643  [doi](http://dx.doi.org/10.1016/j.nbt.2014.12.009)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2015NewBiotechnol_circuits%20for%20biotechnology.pdf)  
<br>  
    

*   **Wang B**\*, Barahona M and Buck M, “Amplification of small molecule-inducible gene expression via tuning of intracellular receptor densities”, **_Nucleic Acids Research_**, 2015, 43(3): 1955-1964  [doi](http://dx.doi.org/10.1093/nar/gku1388)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2015NuclAcidsRes_Amplifying%20gene%20expression%20.pdf)  
<br>  
    

*   **Wang B**\*, Barahona M and Buck M, “Engineering modular and tunable genetic amplifiers for scaling transcriptional signals in cascaded gene networks”, **_Nucleic Acids Research_**, 2014, 42(14): 9484-9492  [doi](http://dx.doi.org/10.1093/nar/gku593)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Nucl.%20Acids%20Res.-2014-Wang_Engineering%20modular%20and%20tunable%20genetic%20amplifiers%20for%20scaling%20transcriptional%20signals%20in%20cascaded%20gene%20networks.pdf)  (Featured in [Imperial College Science News](http://www3.imperial.ac.uk/newsandeventspggrp/imperialcollege/newssummary/news_24-7-2014-17-21-47))  
<br>  
    

*   **Wang B**\* and Buck M, “Rapid engineering of versatile molecular logic gates using heterologous genetic transcriptional modules”, **_Chemical Communications_**, 2014, 50(79): 11642-11644  [doi](http://dx.doi.org/10.1039/C4CC05264A)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2014ChemComm_versatile%20logic%20gates.pdf)  (Featured as Journal [Front Cover Article](http://pubs.rsc.org/en/content/articlepdf/2014/cc/c4cc90362e?page=search); Highlighted as a [Hot ChemComm Article](http://blogs.rsc.org/cc/2014/08/20/hot-chemcomm-articles-for-august/))  
<br>  
    

*   Lawton E, Jovanovic M, Joly N, Waite C, **Wang B**, Burrows P and Buck M, "Determination of the self-association residues within a Homomeric and a Heteromeric AAA + Enhancer Binding Protein", **_Journal of Molecular Biology_**, 2014, 426(8): 1692-1710  [doi](http://dx.doi.org/10.1016/j.jmb.2014.01.001)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Lawton2014JMB_Determination%20of%20the%20Self-Association%20Residues%20within%20AAA%20plus%20EBP.pdf)  
<br>  
    
*   **Wang B**\*, Barahona M and Buck M, “A modular cell-based biosensor using engineered genetic logic circuits to detect and integrate multiple environmental signals”, **_Biosensors and Bioelectronics_**, 2013, 40(1): 368-376  [doi](http://dx.doi.org/10.1016/j.bios.2012.08.011)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2013BiosensorsBioelectronics_biosensor.pdf)  
<br>  
    

*   **Wang B**\*, Barahona M, Buck M and Schumacher J\*, “Rewiring cell signalling through chimeric regulatory protein engineering”, **_Biochemical Society Transactions_**, 2013, 41(5): 1195-1200  [doi](http://www.biochemsoctrans.org/bst/041/1195/bst0411195.htm)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2013BioSocTrans_chimaeric%20protein%20engineering.pdf)  
<br>  
    
*   **Wang B**\* and Buck M\*, “Customizing cell signalling using engineered genetic logic circuits”, **_Trends in Microbiology_**, 2012, 20(8): 376-384  [doi](http://www.sciencedirect.com/science/article/pii/S0966842X12000820)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2012TrendsMicrobiology_cell_signalling.pdf)  
<br>  
    
*   Beguerisse-Diaz M, **Wang B**, Desikan R and Barahona M, “Squeeze-and-breathe evolutionary Monte Carlo Optimisation with local search acceleration and its application to parameter fitting”, **_Journal of the Royal Society Interface_**, 2012, 9(73): 1925-1933  [doi](http://rsif.royalsocietypublishing.org/content/9/73/1925)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Mariano2013JRInterace_optimiation_0.pdf)  
<br>  
    

*   **Wang B**, Kitney R\*, Joly N and Buck M\*, “Engineering modular and orthogonal genetic logic gates for robust digital-like synthetic biology”, **_Nature Communications_**, 2011, 2:508  [doi](http://www.nature.com/ncomms/journal/v2/n10/full/ncomms1516.html)  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2011NatureComms_biological_gates.pdf)  (Reported by _[Financial Times](http://www.ft.com/cms/s/2/25a1f6d4-ff5e-11e0-aa11-00144feabdc0.html#axzz1cjfiNUDt)_, [EPSRC](http://www.epsrc.ac.uk/newsevents/news/computermaygrow/), [European Commission](https://cordis.europa.eu/news/rcn/33946/en), _[Electronics Weekly](http://www.electronicsweekly.com/Articles/18/10/2011/52071/logic-gates-from-bacteria-and-dna.htm)_, _[Science Daily](http://www.sciencedaily.com/releases/2011/10/111018111929.htm)_, [UK Synthetic Biology Roadmap](https://connect.innovateuk.org/documents/2826135/3815409/Synthetic+Biology+Roadmap+-+Report.pdf/fa8a1e8e-cbf4-4464-87ce-b3b033f04eaa) and [Imperial College News](http://www3.imperial.ac.uk/newsandeventspggrp/imperialcollege/newssummary/news_18-10-2011-16-7-29) among others)  
<br>  
    

## 会议论文

*   Ho T and **Wang B**\*, “Intein-assisted Bisection Mapping (IBM)”, **_protocols.io_**, 2020  [doi](https://dx.doi.org/10.17504/protocols.io.bpqdmms6)  
<br>  
    

*   **Wang B**\*, Wan X, Billah M and Rahman A, "Arsenic: smartphone-friendly biosensor to tackle an insidious global threat". _**Microbiologist**_, 2019, 20(3): 24-27  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2019Microbiologist_Arsenic%20-smartphone-friendly%20biosensor%20to%20tackle%20an%20insidious%20global%20threat.pdf)  
<br>  
    

*   **Wang B**\*, "Engineering customised cell signalling circuits and their biotechnological applications". _**New Biotechnology**_, 2014, vol.31:S45  [doi](http://dx.doi.org/10.1016/j.nbt.2014.05.1714)  
<br>  
    

*   Schumacher J, **Wang B**, Bonatto AC and Buck M, “Synthetic transcription factors allow regulon wide control and shifting the nitrogen/carbon balance in bacteria”, **_New Biotechnology,_** 2014, vol.31:S22  [doi](https://doi.org/10.1016/j.nbt.2014.05.1662)  
<br>  
    

*   **Wang B**, Kitney R, Buck M, Jovanovic M, Joly N and James E, “The design and construction of a set of modular synthetic BioLogic devices for programming cells”. World Congress on Medical Physics and Biomedical Engineering, Munich, **_IFMBE Proceedings_**, 2009, vol.25/8, pp.289-292  [doi](http://www.springerlink.com/content/w2vtj3l01577026l/)  
<br>  
    

*   **Wang B**, Bai J, Peng L and Li G, “EEG Recognition Based on Multiple Types of Information by Using Wavelet Packet Transform and Neural Networks”. **_Conf Proc IEEE Eng Med Biol Soc_**. 2005, vol.5:5377-80  [doi](http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=1615697)  
<br>  
    

*   Wang L, Li G, Liu X, **Wang B** and Freeman WJ, “Study of A Chaotic Olfactory Neural Network Model and Its Applications on Pattern Classification”. _**Conf Proc IEEE Eng Med Biol Soc**._ 2005, vol.4:3640-43  [doi](http://dx.doi.org/10.1109/IEMBS.2005.1617270)  
<br>  
    

*   Ding Q, Li G, **Wang B,** Hu M and Li J, “Complexity and Topographic Analysis of EEG under Normal and Simulated High Altitude Acute Hypoxia Conditions”. **_Proceedings of International Conference on Neural Networks and Brain_**, 2005, vol.3:1526-29  [doi](http://dx.doi.org/10.1109/ICNNB.2005.1614922)  
<br>  
    

## 专著及章节

*   Wan X and **Wang B**\*, "Engineering microbial sensor cell arrays to visualize environmental contaminants”, 2021, In: Avraham Rasooly and Houston Baker (eds.), _Biosensors and Biodetection: Methods and Protocols_, Springer; _Methods in Molecular Biology_.  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wan2020BiosensorsBiodetectionChapter_Engineering%20microbial%20sensor%20cell%20arrays%20to%20visualize%20environmental%20contaminants.pdf)  
<br>  
    
*   Wan X, **Ho T** and **Wang B**\*, “Engineering prokaryote synthetic biology biosensors”, 2019, In: Gérald Thouand (ed.) _Handbook of Cell Biosensors_, Springer, 37 pages.  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wan2019BookChapter_Engineering%20prokaryote%20synthetic%20biology%20biosensors.pdf)  [doi](http://dx.doi.org/10.1007/978-3-319-47405-2_131-1)  
<br>  
    
*   Bernard E and **Wang B**\*, "Synthetic cell-based sensors with programmed selectivity and sensitivity”, 2017, In: Avraham Rasooly and Ben Prickril (eds.), _Biosensors and Biodetection: Methods and Protocols_, Springer; _Methods in Molecular Biology_, 1572:349-363  [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Bernard2017MethodsMB_Synthetic%20Cell-based%20Bisoensors%20_Chapter23.pdf)  [doi](http://dx.doi.org/10.1007/978-1-4939-6911-1_23)  
<br>  
    
*   **Wang B**\*, _Engineering Genetic Logic Gates_, Lambert Academic Publishing, ISBN: 978-3-659-27479-4, 2012, 208 pages. [Amazon link](http://www.amazon.co.uk/Engineering-Genetic-Logic-Gates-Functional/dp/3659274798),  [Publisher link](http://www.lap-publishing.com/catalog/details/store/gb/book/978-3-659-27479-4/engineering-genetic-logic-gates?search=Engineering%20Genetic%20Logic%20Gates)  
<br>  
    
*   **Wang B**\*, _Design and Functional Assembly of Synthetic Biological Parts and Devices_, PhD Dissertation, Imperial College London, 2011, 175 pages. [pdf](http://wang.bio.ed.ac.uk/wang/sites/sbsweb2.bio.ed.ac.uk.wang/files/Wang2011PhDThesis_Design%20and%20functional%20assembly%20of%20synthetic%20biological%20parts%20and%20devices_14Feb2011_final%20edited%20version.pdf)  
<br>  